examples/features/authz/client/main.go

```diff
@@ -16,7 +16,8 @@
  *
  */
 
-// Binary client is an example client.
+// Binary client demonstrates how to include authorization credentials in the
+// form of metadata for every RPC for server side validation.
```

Comment:
Nit: s/metadata for every RPC/metadata in every RPC/

Reply:
Done